### DISCUSSION TITLE
Regex feature support -- named capture groups?

### DISCUSSION THREAD

**Discussion by Daemoen** [scandum/tintin#122] [Script help]
I was looking to see, and didn't see anything specific about it...

Does Tintin support named capture groups when using regex?  It definitely supports () when using {{ }}, so figured maybe it would support named groups too.

I had tested using (as an example)

```
#alias {{^q(?<qopt>\w+) ?(\w+)?$}} {#show $qopt}
```

Now I realize I can still use %2 in the above case, but it would be neat if Tintin actually had support for native capture groups and would automatically assign the variable.

**Comment by scandum** [comment]
There's no support for named captures. I'm very hesitant to add redundant functionality at this point.

**Reply by Daemoen** [reply]
That's fair, it was just a curiosity really, I'm currently trying to have Tintin do all of the same things that I do in my normal client so that I can do stuff while I'm on my work computer, just testing the bounds and seeing what all is available during the conversions.

Thanks!
